src/acl_source.f90

```diff
@@ -520,7 +519,7 @@ subroutine Compute_Momentum_Source_Term_pointwise
       enddo
 
       alm_proj_time=MPI_WTIME()-t1
-      call MPI_ALLREDUCE(alm_proj_time,t1,1,real_type,MPI_SUM,MPI_COMM_WORLD,ierr)
+      call MPI_ALLREDUCE(MPI_IN_PLACE,alm_proj_time,1,real_type,MPI_SUM,MPI_COMM_WORLD,ierr)
```

Comment:
This modification is fixing a minor issue ? The code was printing the local value before. Now it is printing the global value obtained after reduction.

Reply:
possibly a typo on my side :-)

Reply:
The other modifications are looking equivalent to the current code. This is probably the only one changing the behavior.

Reply:
I will have a look and fix the typo

Reply:
As a side remark, MPI_Wtime is returning a double precision number. If accuracy is needed on the timer, the single precision builds should not store the result in a variable defined with `real(mytype)` but inside a double precision variable. However, if there is no need for accuracy in the timer, it is ok to store it inside a variable `real(mytype)`

Reply:
I think the new version is better, no need to fix in my opinion